src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionStatusEntity.java

```diff
@@ -7,13 +7,15 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.SequenceGenerator;
 import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 
 @Entity
 @Table(name = "transcription_status")
 @Getter
 @Setter
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Why do we need this annotation added back to 3 of the entities? Is this a mistake?
* TranscriptionStatusEntity
* TranscriptionTypeEntity
* TranscriptionUrgencyEntity

Reply:
They are lookup tables so they are suitable for caching and caching requires equals and hashcode